lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -544,9 +589,13 @@ void TypeLoweringVisitor::visitDecl(WireOp op) {
     std::string loweredName = "";
     if (!name.empty())
       loweredName = name + field.suffix;
+    SmallVector<Attribute> loweredAttrs;
+    // For all annotations on the parent op, filter them based on the target
+    // attribute.
+    filterAnnotations(op.annotations(), loweredAttrs, field, context);
     auto wire = builder->create<WireOp>(field.getPortType(),
                                         builder->getStringAttr(loweredName),
-                                        op.annotations());
+                                        ArrayAttr::get(context, loweredAttrs));
```

Comment:
Why does `WireOp` take an attr, but `RegOp` take an array of annotations?  I think we should standardize on one or the other.

Reply:
Will address this in a separate PR